tfexec/terraform.go

```diff
@@ -129,6 +133,18 @@ func (tf *Terraform) SetAppendUserAgent(ua string) {
 	tf.appendUserAgent = ua
 }
 
+// SetDisablePluginTls sets the TF_DISABLE_PLUGIN_TLS environment variable for
+// Terraform CLI execution.
+func (tf *Terraform) SetDisablePluginTls(disabled bool) {
+	tf.disablePluginTls = disabled
+}
+
+// SetSkipProviderVerify sets the TF_SKIP_PROVIDER_VERIFY environment variable
+// for Terraform CLI execution.
+func (tf *Terraform) SetSkipProviderVerify(skip bool) {
```

Comment:
You can probably just make this return an error for compatibility mismatch that people can just ignore if they want.

Reply:
`TF_SKIP_PROVIDER_VERIFY` was added in v0.10.0-beta2, so no compatibility check needed: https://github.com/hashicorp/terraform/commit/9944ea6886e8d4da1190b93f6d68088b9468c874

Reply:
But it's no longer used in 0.13